core/engine/src/builtins/temporal/plain_month_day/mod.rs

```diff
@@ -118,17 +114,18 @@ impl PlainMonthDay {
         Ok(month_day_to_string(inner, show_calendar))
     }
 }
-impl IsoDateSlots for JsObject<PlainMonthDay> {
-    fn iso_date(&self) -> temporal_rs::iso::IsoDate {
-        self.borrow().data().inner.iso_date()
-    }
-}
 
-impl GetTemporalCalendar for JsObject<PlainMonthDay> {
-    fn get_calendar(&self) -> Calendar {
-        self.borrow().data().inner.get_calendar()
-    }
-}
+// impl IsoDateSlots for JsObject<PlainMonthDay> {
```

Comment:
nit: I think these can just be removed if they are no longer used.

There's probably a good argument to make sure these are removed from `temporal_rs` as well.